spotify/api.h

```diff
@@ -238,7 +238,7 @@ typedef enum sp_availability {
   SP_TRACK_AVAILABILITY_UNAVAILABLE = 0, ///< Track is not available
   SP_TRACK_AVAILABILITY_AVAILABLE   = 1, ///< Track is available and can be played
   SP_TRACK_AVAILABILITY_NOT_STREAMABLE = 2, ///< Track can not be streamed using this account
-  SP_TRACK_AVAILABILITY_BANNED_BY_ARTIST = 3, ///< Track not available on artist's reqeust
```

Comment:
This file is taken directly from libspotify. Thus, I won't accept any changes to it. Please update the PR and I'll merge the rest of your appreciated fixes.